tawazi/dag.py

```diff
@@ -624,3 +625,66 @@ def _handle_exception(self, graph: DiGraphEx, fut: "Future[Any]", id_: IdentityH
 
                 else:
                     raise NotImplementedError(f"Unknown behavior name: {self.behavior}")
+
+    def config_from_dict(self, config: Dict[str, Any]) -> None:
+        """
+        Allows reconfiguring the parameters of the nodes from a dictionary
+        Args:
+            config: the dictionary containing the config
+                    example: {"nodes": {"a": {"priority": 3, "is_sequential": True}}, "max_concurrency": 3}
+
+        Returns:
+
+        """
+        prio_flag = False
+        if "nodes" in config:
+            for node_name, conf_node in config["nodes"].items():
```

Comment:
let's make it possible to use tag and id

Reply:
done